### PR TITLE
[MIG][11.0] Migration of partner_affiliate to v11

### PR DESCRIPTION
**Pull request by BT-rmartin**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check runbot warning:
```
2018-04-12 09:31:05,858 168 WARNING openerp_test odoo.addons.base.ir.ir_ui_view: Error-prone use of @class in view res.partner.form (): use the hasclass(*classes) function to filter elements by their classes
2018-04-12 09:31:05,966 168 WARNING openerp_test odoo.addons.base.ir.ir_ui_view: Error-prone use of @class in view res.partner.form (partner_affiliate.view_partner_form_add_affiliate): use the hasclass(*classes) function to filter elements by their classes
```

File: partner_affiliate/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    """Add relation affiliate_ids."""
+    _inherit = "res.partner"
+
+    child_ids = fields.One2many(domain=[('is_company', '=', False)])
```
Comment (JayVora-SerpentCS) [inline partner_affiliate/models/res_partner.py]: Even though not mandatory, its good to specify string attribute

Comment (JayVora-SerpentCS) [reply partner_affiliate/models/res_partner.py]: I am even in the favour of writing full details of the field like you did for affilate_ids.

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

**Review by rafaelbn — APPROVED** [review]
Tested in runbot functionaly :+1: 

Thanks!

Please @JayVora-SerpentCS @pedrobaeza take a look

**Review by JayVora-SerpentCS — APPROVED** [review]

**Comment by BT-rmartin** [comment]
Two reviewers approved it. In order to be merged, is any other review needed?
@JayVora-SerpentCS @pedrobaeza @rafaelbn

**Pull request merged**